scrapy/extensions/batches.py

```diff
@@ -99,29 +85,20 @@ def should_trigger(self) -> bool:
         return False
 
     def new_batch(self, file: BinaryIO) -> None:
-        """
-        Resets parameter values back to its initial value and increments
-        self.batch_id.
-        """
         self.file = file
         self.start_time = time.time()
         self.item_count = 0
         self.elapsed_seconds = 0
         self.file_size = 0
-        self.batch_id += 1
 
     def _in_seconds(self, duration: str) -> float:
-        """
-        Convert duration string in format: '<HOURS>:<MINUTES>:<SECONDS>' to seconds in float.
-        """
+        # Convert duration string in format: '<HOURS>:<MINUTES>:<SECONDS>' to seconds in float.
         h, m, s = map(float, duration.split(":"))
         duration_in_secs = h * 60 * 60 + m * 60 + s
         return duration_in_secs
 
     def _in_bytes(self, size: str) -> int:
-        """
-        Convert string size in format: '<SIZE><UNIT>' to bytes in integer.
-        """
+        # Convert string size in format: '<SIZE><UNIT>' to bytes in integer.
```

Comment:
These were docstrings of non-interface methods that described what the methods did (not _how_ they did it), so I believe docstrings were appropriate.